paddle/framework/op_desc.proto

```diff
@@ -51,6 +51,9 @@ message OpDesc {
     // type of this Operator, such as "add", "sub", "fc".
     required string type = 3;
 
```

Comment:
Since CosSim_CPU and CosSim_CUDA are different class, we have to add a field like this:

```
required string device_type = 4; 
```